server/src/main/java/org/elasticsearch/index/mapper/VersionFieldMapper.java

```diff
@@ -42,7 +42,7 @@
         public static final VersionFieldType INSTANCE = new VersionFieldType();
 
         private VersionFieldType() {
-            super(NAME, false, false, true, TextSearchInfo.SIMPLE_MATCH_ONLY, Collections.emptyMap());
+            super(NAME, false, false, true, TextSearchInfo.NONE, Collections.emptyMap());
```

Comment:
This seems to have slipped when I double checked all the mappers, maybe it does not have a test so we don't enforce its consistency.

Reply:
Yes, there's no test for VersionFieldMapper - I'll add one as a follow up.

Reply:
thank you